PublicLands.tm2/style.mss

```diff
@@ -1,25 +1,49 @@
 Map { font-directory: url("./fonts"); }
 
-#lands[adminLevel='federal'] {
-    polygon-fill:red;
-}
-#lands[adminLevel='state'] {
-    polygon-fill:green;
-}
-#lands[adminLevel='county'] {
-    polygon-fill:yellow;
-}
-#lands[adminLevel='local'] {
-    polygon-fill:blue;
-}
+@default_text_halo: white;
+@default_text_color: black;
+@default_text_halo_radius: 2;
 
 #lands {
-    text-face-name: 'Source Sans Pro Regular';
-    text-name:'[adminAgency]';
-    text-size: 12;
-    text-placement:point;
-    text-fill:black;
-    text-min-distance:100;
-    text-min-padding:1;
-    text-wrap-width:100;
-}
\ No newline at end of file
+	polygon-opacity:1;	
+	[zoom >= 14] {
+  		text-face-name: 'Noto Sans Regular';
+    	text-name:'[loc_nm]';
```

Comment:
Im not seeing any text ever render, i dont think loc_nm is a field we have. I think it should render `[name]` or if name is not set then `[adminAgency]`